product_docs/docs/efm/4/04_configuring_efm/01_cluster_properties.mdx

```diff
@@ -481,7 +481,10 @@ If necessary, modify these values to suit your business model.
 
 <div id="remote_timeout" class="registered_link"></div>
 
-Use the `remote.timeout` property to specify how many seconds an agent waits for a response from a remote database server (that is, how long a standby agent waits to verify that the primary database is down before performing failover). The `remote.timeout` property value specifies a timeout value for agent-to-agent communication. Other timeout properties in the cluster properties file specify values for agent-to-database communication.
+Use the `remote.timeout` property to limit how many seconds an agent waits for a response from a remote agent or database. Agents only send messages to each other during cluster events. Examples include:
```

Comment:
@EFM-Bobby worth linking the supported scenarios here?

"during cluster events"

->

[during cluster events](https://www.enterprisedb.com/docs/efm/latest/11_supported_scenarios/)